packages/forms/src/widgets/ColumnsWidget/ColumnsWidget.js

```diff
@@ -13,16 +13,17 @@ function getColumnRenderer(type) {
 	return ObjectField;
 }
 
-function Column({ className, schema, formData, onChange, onBlur, registry }) {
-	const Renderer = getColumnRenderer(schema.type);
+function Column(props) {
```

Comment:
( as we spoke, this params are not used in other case than passing to another component, so we skip variable assignements )